inlong-sort/sort-connectors/hbase/src/main/java/org/apache/inlong/sort/hbase/sink/HBaseSinkFunction.java

```diff
@@ -73,30 +73,25 @@
     private final long bufferFlushIntervalMillis;
     private final HBaseMutationConverter<T> mutationConverter;
     private final String inLongMetric;
-
+    /**
+     * This is set from inside the {@link BufferedMutator.ExceptionListener} if a {@link Throwable}
+     * was thrown.
+     *
+     * <p>Errors will be checked and rethrown before processing each input element, and when the
```

Comment:
Please use `<p/>` or `<p></p>` instead of one `<p>`, thanks.

Reply:
DONE